### DISCUSSION TITLE
Stop working in terminal mac os<title>

### DISCUSSION THREAD

**Discussion by darren1192g** [community/community#42570] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

(base) darrenhardwick@darrens-MacBook-Pro ~ % git clone https://github.com/github/debug-repo /tmp/debug-repo-http
git clone git@github.com:github/debug-repo /tmp/debug-repo-ssh
ping -c 10 github.com
traceroute github.com
xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun
xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun
PING github.com (140.82.114.3): 56 data bytes
64 bytes from 140.82.114.3: icmp_seq=0 ttl=49 time=36.865 ms
64 bytes from 140.82.114.3: icmp_seq=1 ttl=49 time=48.866 ms
64 bytes from 140.82.114.3: icmp_seq=2 ttl=49 time=35.731 ms
64 bytes from 140.82.114.3: icmp_seq=3 ttl=49 time=35.177 ms
64 bytes from 140.82.114.3: icmp_seq=4 ttl=49 time=38.940 ms
64 bytes from 140.82.114.3: icmp_seq=5 ttl=49 time=34.283 ms
64 bytes from 140.82.114.3: icmp_seq=6 ttl=49 time=35.351 ms
64 bytes from 140.82.114.3: icmp_seq=7 ttl=49 time=33.949 ms
64 bytes from 140.82.114.3: icmp_seq=8 ttl=49 time=34.904 ms
64 bytes from 140.82.114.3: icmp_seq=9 ttl=49 time=34.233 ms

--- github.com ping statistics ---
10 packets transmitted, 10 packets received, 0.0% packet loss
round-trip min/avg/max/stddev = 33.949/36.830/48.866/4.251 ms
traceroute to github.com (140.82.114.3), 64 hops max, 52 byte packets
 1  10.0.0.1 (10.0.0.1)  5.803 ms  4.446 ms  3.575 ms
 2  96.120.52.93 (96.120.52.93)  16.265 ms  13.367 ms  19.885 ms
 3  ae-252-1209-rur101.murfreesboro.tn.nash.comcast.net (68.85.51.97)  14.379 ms  15.251 ms  14.842 ms
 4  po-101-xar01.murfreesboro.tn.nash.comcast.net (68.85.174.121)  18.330 ms  15.223 ms  17.755 ms
 5  po-2-xar02.murfreesboro.tn.nash.comcast.net (162.151.95.30)  14.464 ms  13.858 ms  15.115 ms
 6  ae-23-ar02.goodslettvll.tn.nash.comcast.net (68.86.148.189)  15.855 ms  16.085 ms  24.099 ms
 7  be-33132-cs03.doraville.ga.ibone.comcast.net (96.110.43.89)  22.813 ms  29.175 ms  23.994 ms
 8  be-1111-cr11.doraville.ga.ibone.comcast.net (96.110.34.162)  26.921 ms
    be-1411-cr11.doraville.ga.ibone.comcast.net (96.110.34.174)  23.136 ms
    be-1111-cr11.doraville.ga.ibone.comcast.net (96.110.34.162)  23.265 ms
 9  be-302-cr13.56marietta.ga.ibone.comcast.net (96.110.39.49)  24.500 ms
    be-303-cr13.56marietta.ga.ibone.comcast.net (96.110.39.221)  23.380 ms
    be-304-cr13.56marietta.ga.ibone.comcast.net (96.110.39.169)  22.819 ms
10  be-1313-cs03.56marietta.ga.ibone.comcast.net (96.110.34.233)  25.240 ms
    be-1213-cs02.56marietta.ga.ibone.comcast.net (96.110.34.229)  24.256 ms  23.456 ms
11  be-2412-pe12.56marietta.ga.ibone.comcast.net (96.110.33.190)  23.276 ms
    be-2212-pe12.56marietta.ga.ibone.comcast.net (96.110.33.182)  22.467 ms
    be-2312-pe12.56marietta.ga.ibone.comcast.net (96.110.33.186)  24.747 ms
12  50.248.116.42 (50.248.116.42)  26.421 ms  23.357 ms  23.016 ms
13  ash-bb2-link.ip.twelve99.net (62.115.125.129)  38.728 ms  50.761 ms  36.575 ms
14  ash-b2-link.ip.twelve99.net (62.115.123.125)  44.056 ms  39.865 ms  37.728 ms
15  github-ic345016-ash-b2.ip.twelve99-cust.net (62.115.175.101)  40.530 ms
    github-ic345015-ash-b2.ip.twelve99-cust.net (62.115.175.99)  38.531 ms  40.815 ms
16  * * *
17  * * *
18  * * *
19  * * *
20  * * *
21  * * *
22  * * *
23  * * *
24  * * *
25  * * *
26  * * *
27  * * *
28  * * *
29  * * *
30  * * *
31  * * *
32  * * *
33  * * *
34  * * *
35  * * *
36  * * *
37  * * *
38  * * *
39  * * *
40  * * *
41  * * *
42  * * *
43  * * *
44  * * *
45  * * *
46  * * *
47  * * *
48  * * *
49  * * *
50  * * *
51  * * *
52  * * *
53  * * *
54  * * *
55  * * *
56  * * *
57  * * *
58  * * *
59  * * *
60  * * *
61  * * *
62  * * *
63  * * *
64  * * *

**Comment by jfullstackdev** [comment]
you can also ask there the same question, create an Issue to do so
